e2e/connect/input/demo.nomad

```diff
@@ -0,0 +1,61 @@
+ job "countdash" {
+   datacenters = ["dc1"]
+
+   group "api" {
+     network {
+       mode = "bridge"
+     }
+
+     service {
```

Comment:
Do you think we could conjure up a script check for these using curl?

Reply:
Let's save it for a future e2e test. Websockets make this one hard to deal with.